massa-grpc/src/stream/mod.rs

```diff
@@ -0,0 +1,11 @@
+/// send_blocks streaming
+pub mod send_blocks;
```

Comment:
I don't know If It's a good design to create a file for every stream ? may be regroup methods by domain ?